src/MarketplaceHandler.ts

```diff
@@ -68,6 +76,23 @@ export function handlePodListingCreated(event: PodListingCreated): void {
     marketDaily.blockNumber = event.block.number
     marketDaily.timestamp = event.block.timestamp
     marketDaily.save()
+
+    // Save the raw event data
+    let id = 'podListingCreated-' + event.transaction.hash.toHexString() + '-' + event.logIndex.toString()
+    let rawEvent = new PodListingCreatedEvent(id)
```

Comment:
From an implementation standpoint, think this is good to have raw data. Need to discuss with @imcoolbean how the UI will parse & implement the history view. For now let's roll with this.